templates/index.html

```diff
@@ -63,11 +63,11 @@ <h3>Wikis</h3>
   {% for wiki_name, wiki_data in wikis|dictsort %}
     <tr><td>{{ wiki_name }}</td>
 <td>{% if ( wiki_data['models'] and wiki_data['models']['reverted'] ) %}<a href="https://ores.wikimedia.org/v3/scores/{{wiki_name}}"><img alt="Yes check.svg" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/fb/Yes_check.svg/10px-Yes_check.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/fb/Yes_check.svg/15px-Yes_check.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/fb/Yes_check.svg/20px-Yes_check.svg.png 2x" data-file-width="600" data-file-height="600" width="10" height="10"></a>{% else %}{% if ( wiki_data['models'] and wiki_data['models']['damaging'] ) %}n/a{% endif %}{% endif %}</td>
-<td>{% if ( wiki_data['campaigns'] and wiki_data['campaigns']['damaging_and_goodfaith'] ) %}<small><a href="https://labels.wmflabs.org/stats/{{wiki_name}}/{{wiki_data['campaigns']['damaging_and_goodfaith']['id']}}">{{ '%d' | format((wiki_data['campaigns']['damaging_and_goodfaith']['done'] * 100) /wiki_data['campaigns']['damaging_and_goodfaith']['total']|float) }}%</a></smal>{% endif %}</td>
+<td{% if ( wiki_data['campaigns'] and wiki_data['campaigns']['damaging_and_goodfaith'] and ((wiki_data['campaigns']['damaging_and_goodfaith']['done'] * 100) /wiki_data['campaigns']['damaging_and_goodfaith']['total']) > 90 ) %} style="background: #d5fdf4;"{% endif %}>{% if ( wiki_data['campaigns'] and wiki_data['campaigns']['damaging_and_goodfaith'] ) %}<small><a href="https://labels.wmflabs.org/stats/{{wiki_name}}/{{wiki_data['campaigns']['damaging_and_goodfaith']['id']}}">{{ '%d' | format((wiki_data['campaigns']['damaging_and_goodfaith']['done'] * 100) /wiki_data['campaigns']['damaging_and_goodfaith']['total']|float) }}%</a></smal>{% endif %}</td>
```

Comment:
All of this logic should be moved out of the template and into a Python function that builds template parameters.

Reply:
Line breaks!